scripts/run_backend_tests.sh

```diff
@@ -85,14 +85,6 @@ for arg in "$@"; do
   fi
 done
 
-# Compile typescript files
-echo "Compiling typescript..."
-$NODE_MODULE_DIR/typescript/bin/tsc --project .
-
-echo "Compiling webpack..."
-$NODE_MODULE_DIR/webpack/bin/webpack.js --config webpack.prod.config.ts
-$PYTHON_CMD scripts/build.py
-
```

Comment:
Could you do this in separate PR so it can get into the codebase quicker, I will try to approve it as quick as possible.

Reply:
This wasn't an expected change. I was trying to run the backend test quickly on the test server to get the result.

This PR contains structural migration and It was hard for me to run the entire backend test locally.

I've removed this changes!